### DISCUSSION TITLE
Next.js project + Google Cloud Mysql using Vercel Integrations

### DISCUSSION THREAD

**Discussion by ralvs** [vercel/vercel#4882] [Help]
Hello guys!
I just would like to confirm if I did this the right way...

I have successfully deployed my project to Vercel and enable the Google Integration

<img width="749" alt="Screen Shot 2020-07-18 at 17 42 19" src="https://user-images.githubusercontent.com/5287233/87861600-3abeb500-c91e-11ea-9477-5eb92ff0000c.png">
<img width="563" alt="Screen Shot 2020-07-18 at 17 41 55" src="https://user-images.githubusercontent.com/5287233/87861602-3eead280-c91e-11ea-8c8f-ea2b8b8fc10d.png">

But after this, the communication between then only work if I add 0.0.0.0/0 as a allowed network in GCloud configurations.

Is that correct?

I was wondered that after enabling the integration and telling google that vercel is a owner of the project, everything should magically works :)

Thanks

**Comment by paulkarayan** [comment]
i think this is the only way to do it, because Vercel (or netlify for that matter) won't give you a static ip or ip range to whitelist.
